Nodejs/Product/TestAdapter/JavaScriptTestDiscoverer.cs

```diff
@@ -60,31 +70,56 @@ public void DiscoverTests(IEnumerable<string> sources, IDiscoveryContext discove
                     {
                         var projectHome = Path.GetFullPath(Path.Combine(proj.DirectoryPath, "."));
 
-                        var testItems = new Dictionary<string, List<TestFileEntry>>(StringComparer.OrdinalIgnoreCase);
+                        var testItems = new Dictionary<string, HashSet<TestFileEntry>>(StringComparer.OrdinalIgnoreCase);
```

Comment:
>HashSet [](start = 63, length = 7)

Why did this change?  To eliminate duplicates?

Reply:
Yes